api/http/handler/websocket/websocket_exec.go

```diff
@@ -167,19 +167,30 @@ func createDial(endpoint *portainer.Endpoint) (net.Conn, error) {
 	var host string
```

Comment:
I think that we should default to `url.Host` here by assuming `url.Scheme == tcp`.

This simplifies the if/else statement below.

```go
host := url.Host

if url.Scheme == "unix" || url.Scheme = "npipe" {
host = url.Path
}
```